recipes/pypolyagamma-3/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+if [ $(uname) == Darwin ]
+then
+    export CC=gcc CXX=g++
+fi
+
+if [ $(uname) == Linux ]
+then
+    yum install gsl-devel
```

Comment:
> Here it is, I'll post it on gsl-feedstock also.

No need as that is unrelated to the `gsl` package. You will need to tell `pypolyagamma-3` setup where to find the libraries since they are in a non-standard place in conda. If `setup.py` is "smart" you can try:

`$PYTHON setup.py build_ext -I$PREFIX/include install --single-version-externally-managed --record record.txt`

(Maybe also need `-L$PREFIX/lib -lgsl` ?)